src/config.js

```diff
@@ -18,8 +18,18 @@
 
 module.exports = {
   ios: {
-    globs: ['**/*.{m,h}'],
-    patterns: [/#import <Backpack\/[a-zA-Z]+(\/[a-zA-Z]+)?\.h>/g],
+    globs: ['**/{Modules,App}/**/*.{m,swift}', '!**/{Tests,UITests}/**'],
+    patterns: [
+      /BPKColor/gi,
+      /BPKFont/gi,
+      /BPKSpacing/gi,
+      /BPKBorderRadius/gi,
+      /BPKShadow/gi,
+      /BPKGradient/gi,
+      /BPKBadge/gi,
+      /BPKLabel/gi,
+      /BPKPanel/gi,
+    ],
```

Comment:
So would this now capture multiple uses of a component in one file, whereas before it would only work at most once per component per file, because it was using imports?

Reply:
It's very rudimentary, we check if the files contents contains at least one of those know symbols .. if it does we determine that to be "usage".

Reply:
We also have `BPKGradientView`, `BPKGradientLayer`, and `BPKIcon`

Reply:
@k0nserv can you think of any downsides to changing the pattern to just `/BPK/gi`? Just thinking it'll be more maintainable if we don't have to keep updating this config file whenever we make new components.